Mtneer/mushu
# View all categories

# Description
> As an admin I would like to see all the available categories so that I can choose to edit or delete one, or see that I should add a new one.
> 
> **Given** an admin is in the app
> **When** they select the `Category Management` link in the menu
> **Then** they should be directed to a page that lists all the Category names ordered alphabetically

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- pull down new branch
- start the web api and react app
- login to the app using admin@admin.com and password1
- click on the categories tab
- ensure that a list of categories appear in alphabetical order

# Checklist:
